Undefined symbol "__svm_vm_is_static_binary"

With latest GraalVM 20.3.0-dev/20.2.0-dev builds (7th August).

Follow-up of https://github.com/gluonhq/client-samples/issues/103

If fails linking on iOS as well:
```
[dom. ago. 09 13:12:24 CEST 2020][INFO] [SUB] Undefined symbols for architecture arm64:
[dom. ago. 09 13:12:24 CEST 2020][INFO] [SUB]   "___svm_vm_is_static_binary", referenced from:
[dom. ago. 09 13:12:24 CEST 2020][INFO] [SUB]       _JVM_FindLibraryEntry in libjvm.a(JvmFuncs.o)
[dom. ago. 09 13:12:25 CEST 2020][INFO] [SUB] ld: symbol(s) not found for architecture arm64
```

--- a/src/main/resources/native/android/c/launcher.c
+++ b/src/main/resources/native/android/c/launcher.c
@@ -50,6 +50,8 @@ int start_logger(const char *app_name);
 // TODO: remove once https://github.com/oracle/graal/issues/2713 is fixed
 int JNI_OnLoad_sunec(JavaVM *vm, void *reserved);
 
+int __svm_vm_is_static_binary = 1;
+
 const char *origargs[] = {
     "myapp",
     "-Djavafx.platform=android",

--- a/src/main/resources/native/android/c/launcher.c
+++ b/src/main/resources/native/android/c/launcher.c
@@ -50,7 +50,7 @@ int start_logger(const char *app_name);
 // TODO: remove once https://github.com/oracle/graal/issues/2713 is fixed
 int JNI_OnLoad_sunec(JavaVM *vm, void *reserved);
 
-int __svm_vm_is_static_binary = 1;
+extern int __svm_vm_is_static_binary __attribute__((weak)) = 1;
 
 const char *origargs[] = {
     "myapp",

--- a/src/main/resources/native/ios/AppDelegate.m
+++ b/src/main/resources/native/ios/AppDelegate.m
@@ -57,6 +57,8 @@ int startGVM(const char* userHome, const char* userTimeZone);
 // TODO: remove once https://github.com/oracle/graal/issues/2713 is fixed
 int JNI_OnLoad_sunec(JavaVM *vm, void *reserved);
 
+int __svm_vm_is_static_binary = 1;
+
 extern int *run_main(int argc, const char* argv[]);
 
 @interface AppDelegate ()

--- a/src/main/resources/native/ios/AppDelegate.m
+++ b/src/main/resources/native/ios/AppDelegate.m
@@ -57,7 +57,7 @@ int startGVM(const char* userHome, const char* userTimeZone);
 // TODO: remove once https://github.com/oracle/graal/issues/2713 is fixed
 int JNI_OnLoad_sunec(JavaVM *vm, void *reserved);
 
-int __svm_vm_is_static_binary = 1;
+extern int __svm_vm_is_static_binary __attribute__((weak)) = 1;
 
 extern int *run_main(int argc, const char* argv[]);
 

